doc/source/whatsnew/v0.25.0.rst

```diff
@@ -77,6 +77,40 @@ is respected in indexing. (:issue:`24076`, :issue:`16785`)
 
     df['2019-01-01 12:00:00+04:00':'2019-01-01 13:00:00+04:00']
 
+
+.. _whatsnew_0250.api_breaking.multi_indexing:
+
+
+MultiIndexing contracted from levels and codes
```

Comment:
I don't think "contracted" is the right word. Maybe "Changed allowed codes in MultiIndex"?

Reply:
Changed the typo in whatsnew, I think I meant constructed